AIMS/Program.cs

```diff
@@ -1,10 +1,24 @@
+using AIMS.Data;
+using Microsoft.EntityFrameworkCore;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddControllersWithViews();
+builder.Services.AddEndpointsApiExplorer(); // take out after development
+builder.Services.AddSwaggerGen(); // take out after development

```

Comment:
Thank you for integrating Swagger: it's been a great experience in my experience.